src/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { store } from './app/store';
+import { Provider } from 'react-redux';
+import * as serviceWorker from './serviceWorker';
+import App from './App';
+
+ReactDOM.render(
```

Comment:
ReactDOM.render is deprecated in React 18, replace with root.render.

const rootElement = document.getElementById("root") as HTMLElement;
    const root = createRoot(rootElement);

    root.render(
    <React.StrictMode>
        <Provider store={store}>
        <App />
        </Provider>
    </React.StrictMode>
    );